### DISCUSSION TITLE
board weact studio stm32f405rgt6

### DISCUSSION THREAD

**Discussion by domenic64** [micropython/micropython#15386] [STM32 / Pyboard]
Hi, I purchased a weact studio stm32f405rgt6 card to use it with micropython after loading the PYBV10-20240602-v1.23.0.dfu file in dfu mode without errors, I perform a reset and the card does not display any USB connection on the PC, there is need special firmware.
Thank you

**Comment by scruss** [comment]
You might need to build your own using WeActStudio's definitions: [WeActStudio.STM32F4_64Pin_CoreBoard/Examples/STM32F405/mpy at master · WeActStudio/WeActStudio.STM32F4_64Pin_CoreBoard](https://github.com/WeActStudio/WeActStudio.STM32F4_64Pin_CoreBoard/tree/master/Examples/STM32F405/mpy).

I see there's a hex file for a MicroPython there too, but it's 1.20

**Reply by domenic64** [reply]
Ok It works for now with versione 1.20
Thank you
